app/src/main/res/layout/topic_overview_fragment.xml

```diff
@@ -1,32 +1,51 @@
 <?xml version="1.0" encoding="utf-8"?>
-<layout
-  xmlns:android="http://schemas.android.com/apk/res/android"
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto">
+
   <data>
+
     <variable
       name="presenter"
-      type="org.oppia.app.topic.overview.TopicOverviewFragmentPresenter"/>
+      type="org.oppia.app.topic.overview.TopicOverviewFragmentPresenter" />
+
     <variable
       name="viewModel"
-      type="org.oppia.app.topic.overview.TopicOverviewViewModel"/>
+      type="org.oppia.app.topic.overview.TopicOverviewViewModel" />
   </data>
+
   <ScrollView
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@color/white"
     android:overScrollMode="never"
     android:scrollbars="none">
+
     <androidx.constraintlayout.widget.ConstraintLayout
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:paddingBottom="24dp">
-      <ImageView
-        android:id="@+id/topic_thumbnail_image_view"
-        android:layout_width="0dp"
-        android:layout_height="200dp"
-        android:src="@{viewModel.topic.topicThumbnail.thumbnailGraphic}"
+
+      <FrameLayout
+        android:id="@+id/topic_thumbnail_frame_layout"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"/>
+        app:layout_constraintTop_toTopOf="parent">
+
+        <ImageView
+          android:id="@+id/topic_thumbnail_image_view"
+          android:layout_width="match_parent"
+          android:layout_height="202dp"
```

Comment:
Not a multiple of 4.

Reply:
As per the conversation thread, I am still not able to understand how we have finalised this?